db/migrate/20160401173409_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string  :content
+      t.integer :toilet_id
```

Comment:
does not need toilet_id as a field. This is managed by the belongs_to in ActiveRecord